paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/ActionFactory.java

```diff
@@ -151,8 +151,8 @@ static Map<String, String> parseCommaSeparatedKeyValues(String keyValues) {
     }
 
     static void parseKeyValueString(Map<String, String> map, String kvString) {
-        String[] kv = kvString.split("=");
-        if (kv.length != 2) {
+        String[] kv = kvString.split("=", 2);
+        if (kv.length < 2) {
```

Comment:
Why != to < ?

Reply:
because we add the `limit` for `split` method, the max length of `kv` array is 2,  the exception is that the expression does not have `=`,  which means that `kv.length < 2`

Reply:
`kv.length != 2` Where is the difference?

Reply:
`!= 2` is also correct. What I started thinking was that after adding a limit 2 , there would be no situation `>2`, so I changed it to `<2`

I rollback the code.